opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -569,13 +570,13 @@ def _from_env_if_absent(
         return value
 
 
-_UnsetLimits = _Limits(
-    max_attributes=_Limits.UNSET,
-    max_events=_Limits.UNSET,
-    max_links=_Limits.UNSET,
+_UnsetLimits = SpanLimits(
+    max_attributes=SpanLimits.UNSET,
+    max_events=SpanLimits.UNSET,
+    max_links=SpanLimits.UNSET,
```

Comment:
What do you think about making this public? If I wanted to unset all limits in code I would have do exactly same.

Reply:
Not sure. You'd have to type it all but it'd a one time thing per service so I'm not that concerned. I'd prefer to keep as many things private by default as possible. We can always make it public later but not the other way round :)

Reply:
That is fair. I am concerned about the growing surface area and number of concepts one needs to learn to setup a pipeline. Can you add some usage examples (preferably copy/paste-able) in SpanLimits docstring?

Reply:
That is valid but it is totally optional. We provide sane defaults so people only need to learn this if the defaults don't work for them.